lib/_tls_common.js

```diff
@@ -249,3 +251,13 @@ exports.translatePeerCertificate = function translatePeerCertificate(c) {
   }
   return c;
 };
+
+exports.derCertToPemCert = function derCertToPemCert(cert) {
```

Comment:
On a quick glance, it seems fairly trivial to implement this without adding dependencies to your application? Why won't someone just write this function out?